addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -160,7 +160,7 @@ function clickOnElement(elementName, selector) {
  */
 function clickOnEditAndWaitEditMode(position = "bottom") {
     return [{
-        content: _t("<b>Click Edit</b> to start designing your homepage."),
+        content: Markup(_t("<b>Click Edit</b> to start designing your homepage.")),
```

Comment:
you should use the new `markup` function exported by @odoo/owl directly, it would ease the forward port of this PR, as `Markup` has been removed in the next versions

Reply:
Hello @aab-odoo 
I have made changes as per your suggestion. Please look at it.

Thanks!